Rakefile

```diff
@@ -209,8 +209,8 @@ task :check_consistency_between_aws_and_carrenza do
     govuk::apps::email_alert_api::db_password
     govuk::apps::email_alert_api::delivery_request_threshold
     govuk::apps::email_alert_api::email_address_override
-    govuk::apps::email_alert_api::email_address_override_whitelist
-    govuk::apps::email_alert_api::email_address_override_whitelist_only
+    govuk::apps::email_alert_api::allowed_email_addresses
+    govuk::apps::email_alert_api::allowed_email_addresses_only
```

Comment:
This is one confusing boolean (the original, that is). Looking in [the code](https://github.com/alphagov/email-alert-api/blob/4890e19da81d19a91fdafed783a0ffcd9f907b3d/app/services/delivery_request_service/email_address_overrider.rb), it seems its purpose is to:

- If `true`, only send email to the configured addresses
- If `false`, send all other email to an "override address"

I think we could just scrap this variable, and just rely on the presence of the override address if one is set.

Reply:
It appears to be used for logic [here](https://github.com/alphagov/govuk-puppet/pull/10329/files/674788995ba1e2abf377923d15041e23e1b935c5#diff-b1c1d8bf6c96caf2e78a19d9bdbbf71bR264)